utils/api.py

```diff
@@ -79,21 +79,22 @@ def tasks_create_file():
     platform = request.forms.get("platform", "")
     tags = request.forms.get("tags", None)
     custom = request.forms.get("custom", "")
-    memory = request.forms.get("memory", False)
+    memory = request.forms.get("memory", 'False')
     clock = request.forms.get("clock", None)
     shrike_url = request.forms.get("shrike_url", None)
     shrike_msg = request.forms.get("shrike_msg", None)
     shrike_sid = request.forms.get("shrike_sid", None)
     shrike_refer = request.forms.get("shrike_refer", None)
-    
-    if memory.isdigit() and int(memory):
-        memory = True
-    elif memory:
+
+    if memory.upper() == 'FALSE':
+        memory = False
+    else:
         memory = True
-    enforce_timeout = request.forms.get("enforce_timeout", False)
-    if enforce_timeout.isdigit() and int(enforce_timeout):
-        enforce_timeout = True
-    elif enforce_timeout:
+
+    enforce_timeout = request.forms.get("enforce_timeout", 'False')
+    if enforce_timeout.upper() == 'FALSE':
+        enforce_timeout = False
+    else:
```

Comment:
this will break dist.py for new, you can just do this check in elif

also elif will handle that default False value correctly, i don't know why do you need convert it yo string